src/runtime/pythonengine.cs

```diff
@@ -389,15 +413,9 @@ public static PyObject ReloadModule(PyObject module)
         public static PyObject ModuleFromString(string name, string code)
         {
             IntPtr c = Runtime.Py_CompileString(code, "none", (IntPtr)257);
-            if (c == IntPtr.Zero)
-            {
-                throw new PythonException();
-            }
+            Py.Throw();
             IntPtr m = Runtime.PyImport_ExecCodeModule(name, c);
-            if (m == IntPtr.Zero)
-            {
-                throw new PythonException();
-            }
+            Py.Throw();
```

Comment:
Because this function either returns null and sets an exception, or it returns a module.